src/js/page.js

```diff
@@ -200,8 +204,8 @@
             window.sessionStorage['_sidecar_debug_injection_timeout'] = String(ms);
         },
 
-        console: function(data) {
-            console.log(data);
+        console: function(level, data) {
```

Comment:
no, we were using it only once [here](https://github.com/sugarcrm/SidecarDevTools/pull/42/files#diff-36fdf154dd2c69b71f94e9e658d1a1b0R258) and I updated it